src/app/admin/store/selectors/admin-book.selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { AppState } from '../../../store/state/app.state';
+import { State } from '../reducers';
+import { AdminBookState } from '../state/admin-books.state';
+
+export const selectAdmin = createFeatureSelector<AppState, State>('admin');
```

Comment:
optional: it's great idea to divide selectors with empty lines between them
help a lot with readability

Reply:
fixed